### DISCUSSION TITLE
VHF Contests with serial number and 6 digit locator

### DISCUSSION THREAD

**Discussion by scotthibbs** [w7sst/MorseRunner#102] [New Contest Suggestions]
Pascal F5LEN asks for this on twitter here: https://twitter.com/f5len/status/1582071905481170944?s=20&t=V7bAciyx9IT53X8Oj_4CGw

**Comment by w7sst** [comment]
Thank you for including feedback from F5LEN.

Please convert this to a Feature Request or Contest Request Issue. We added a New Feature label called "Contest Request". This is a good example of a feature request.

**Reply by f5gkw** [reply]
Hello, 
Please, I also would like to have a such contest with West EU, or EU grid locator and serial numbers.
Thank you 73 Jean-Phi F5GKW
